### PR TITLE
[AAE-3514] Create file viewer widget

### PR DESCRIPTION
**Pull request by pmartinezga**
**Please check if the PR fulfills these requirements**

> - [X] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [X] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [X] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
https://issues.alfresco.com/jira/browse/AAE-3514

**What is the new behaviour?**
https://issues.alfresco.com/jira/browse/AAE-3514

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [X] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/process-services-cloud/src/lib/form/components/widgets/attach-file/attach-file-cloud-widget.component.ts [hunk 1]
```diff
@@ -152,20 +172,34 @@ export class AttachFileCloudWidgetComponent extends UploadCloudWidgetComponent i
...
-    onRetrieveFileMetadata(file: Node) {
-        const values: FormValues = {};
-        const metadata = file?.properties;
-        if (metadata) {
-            const keys = Object.keys(metadata);
-            keys.forEach(key => {
-                const sanitizedKey = key.replace(':', '_');
-                values[sanitizedKey] = metadata[key];
+    contentModelFormFileHandler(file?: Node) {
+        if (file?.id && this.field?.params?.menuOptions && this.field.params.menuOptions[AttachFileCloudWidgetComponent.RETRIEVE_METADATA_OPTION]) {
```
Comment (davidcanonieto) [inline lib/process-services-cloud/src/lib/form/components/widgets/attach-file/attach-file-cloud-widget.component.ts:188]: can we have this condition as a function to make the code easier to read? `isFileWhatever(file)`

Comment (pmartinezga) [reply lib/process-services-cloud/src/lib/form/components/widgets/attach-file/attach-file-cloud-widget.component.ts]: sure 👍

File: lib/process-services-cloud/src/lib/form/components/widgets/attach-file/attach-file-cloud-widget.component.scss [hunk 2]
```diff
@@ -65,9 +64,23 @@
...
+
         .mat-line {
             margin-bottom: 0;
         }
     }
 
+    &-attach-selected-file-row {
+        div.mat-list-item-content {
+            .adf-datatable-selected {
+                color: #00bcd4 !important;
```
Comment (eromano) [inline lib/process-services-cloud/src/lib/form/components/widgets/attach-file/attach-file-cloud-widget.component.scss:81]: this color shoudl come from the theme

File: lib/process-services-cloud/src/lib/form/components/widgets/attach-file/attach-file-cloud-widget.component.ts [hunk 3]
```diff
@@ -152,20 +172,38 @@ export class AttachFileCloudWidgetComponent extends UploadCloudWidgetComponent i
...
+                    });
+                    this.formService.updateFormValuesRequested.next(values);
+                }
             });
-            this.formService.updateFormValuesRequested.next(values);
         }
+        this.fileClicked(new UploadWidgetContentLinkModel(file, this.field.id));
+    }
+
+    isRetrieveMetadataOptionEnabled() {
```
Comment (eromano) [inline lib/process-services-cloud/src/lib/form/components/widgets/attach-file/attach-file-cloud-widget.component.ts:205]: missing return type

**Review by eromano — CHANGES_REQUESTED** [review]

**Review by eromano — APPROVED** [review]

**Pull request merged**